health/micro-ui/web/micro-ui-internals/packages/modules/health-dss/src/components/Icon.js

```
@@ -0,0 +1,71 @@
+import React from "react";
+import { CustomSVG } from "@egovernments/digit-ui-components";
+
+const Bednet = ({ width = 26, height = 32, fill = "#C84C0E", className }) => (
+  <svg xmlns="http://www.w3.org/2000/svg" width={width} height={height} viewBox="0 0 26 32" fill="none" className={className}>
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)